### DISCUSSION TITLE
开源参与度高校排行榜

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#117] [2️⃣ 聊项目]
结合今年的几个暑期开源编程活动，统计相关数据做一个高校的开源参与度排行榜：

**GSoC**：https://summerofcode.withgoogle.com/
**OSPP**：https://summer-ospp.ac.cn/
**ASoC**：https://open-source.alibaba.com/blog/detail/73
**GLCC**：https://www.gitlink.org.cn/glcc/result
**LXM**：https://lfx.linuxfoundation.org/tools/mentorship/

通过**高校开源参与排行榜**，吸引大家在 OpenDigger 下面对学校和学校的学生贡献者打标签，进而推出类似 OpenLeadboard 的**高校开源贡献/影响力排行榜**~

**Comment by xgdyp** [comment]
OSPP:在OSPP官网上有[数据展示页](https://summer-ospp.ac.cn/#/data)，里面包含了学校对应的申请情况。对其数据包进行简单的处理可以拿到对应的数据，这里我贴到了[石墨文档中](https://shimo.im/sheets/1d3aVovVNdfP0zqg/MODOC/ospp_data)

**Comment by xgdyp** [comment]
ASoC:https://shimo.im/sheets/5xkGMPKo5nFROE3X/MODOC

**Comment by xgdyp** [comment]
GLCC：https://shimo.im/sheets/RKAWVGK6OdSx9Wk8/MODOC

**Comment by will-ww** [comment]
Many thanks, 我把这个也作为设计思维课程的一个候选项目了，看看有没人感兴趣做一个，例如借助 DataV 大屏做一个~ @xgdyp

**Reply by xgdyp** [reply]
明白了～

**Comment by will-ww** [comment]
![image](https://user-images.githubusercontent.com/15010826/179489254-8203c412-93f8-49e3-86a7-c2f656ffbb9c.png)
